src/tests/multihost/basic/conftest.py

```diff
@@ -173,6 +175,117 @@ def disable_kcm():
     request.addfinalizer(disable_kcm)
 
 
+@pytest.fixture
+def create_casesensitive_posix_user(session_multihost):
+    """ Create a case sensitive posix user """
+    ldap_uri = 'ldap://%s' % (session_multihost.master[0].sys_hostname)
+    krb = krb5srv(session_multihost.master[0], 'EXAMPLE.TEST')
+    ds_rootdn = 'cn=Directory Manager'
+    ds_rootpw = 'Secret123'
+    ldap_inst = LdapOperations(ldap_uri, ds_rootdn, ds_rootpw)
+    username = 'CAPSUSER-1'
+    user_info = {'cn': username,
+                 'uid': username,
+                 'uidNumber': '24583100',
+                 'gidNumber': '14564100'}
+    ldap_inst.posix_user("ou=People", "dc=example,dc=test", user_info)
+    krb.add_principal('CAPSUSER-1', 'user', 'Secret123')
+
+
+@pytest.fixture
+def set_case_sensitive_false(session_multihost):
+    """ Set case_sensitive to false in sssd domain section """
+    session_multihost.master[0].transport.get_file('/etc/sssd/sssd.conf',
+                                                   '/tmp/sssd.conf')
+    sssdconfig = ConfigParser.SafeConfigParser()
+    sssdconfig.read('/tmp/sssd.conf')
+    domain_section = "%s/%s" % ('domain', 'EXAMPLE.TEST')
+    if domain_section in sssdconfig.sections():
+        sssdconfig.set(domain_section, 'case_sensitive', 'false')
+        with open('/tmp/sssd.conf', "w") as sssconf:
+            sssdconfig.write(sssconf)
+    session_multihost.master[0].transport.put_file('/tmp/sssd.conf',
+                                                   '/etc/sssd/sssd.conf')
+    session_multihost.master[0].service_sssd('restart')
+
+
+@pytest.fixture
+def enable_sudo_service(session_multihost):
+    """ Enable sudo services in sssd.conf """
+    session_multihost.master[0].transport.get_file('/etc/sssd/sssd.conf',
+                                                   '/tmp/sssd.conf')
+    sssdconfig = ConfigParser.SafeConfigParser()
+    sssdconfig.read('/tmp/sssd.conf')
+    sssdconfig.remove_option('sssd', 'services')
+    sssdconfig.set('sssd', 'services', 'nss, pam, ifp, sudo')
```

Comment:
I think there is no downside to always enabling the sudo service. If it's not running, the sudo client piece will handle this just fine.

Reply:
okay i can enable it by default. in the current fixture, that way it's one less fixture.